.travis.yml

```diff
@@ -35,14 +35,18 @@ matrix:
       compiler: clang
       env:
         SCAN_BUILD=yes
-        CMAKE_FLAGS=-DGOOGLE_CLOUD_CPP_GRPC_PROVIDER=package
-        DISTRO=ubuntu-with-dependencies
+        DISTRO=ubuntu
         DISTRO_VERSION=16.04
+      # Because this build installs gRPC we can skip that submodule, that
```

Comment:
- s/gRPC we/gRPC, we/
- s/submodule, that saves/submodule, which saves us/

Similarly everywhere else this same sentence appears.

Reply:
Fixed in both places.